### ISSUE TITLE
Bug: module.path is not defined in NodeJS < 11.14.0

### ISSUE THREAD

**Issue by webketje** [AllMightySauron/ascii-table3#1]
Hey, nice pkg, although it doesn't work out-of-the-box.
Offending line is: https://github.com/AllMightySauron/ascii-table3/blob/main/ascii-table3.js#L79

If you change
```js
fs.readFileSync(module.path ? module.path + '/' + STYLES_FILENAME : STYLES_FILENAME, 'utf8')
```
to 
```js
this.styles = require(path.join(__dirname, STYLES_FILENAME));
```
it works a charm and will always resolve correctly

**Comment by AllMightySauron** [comment]
Hi @webketje,

Just trying to understand the specific configuration where it is not working. Are you using the package standalone in node.js or inside a browser?

**Comment by webketje** [comment]
@AllMightySauron trialing it as a dependency for a Node.JS package like 
```js
const { AsciiTable3 } = require('ascii-table3');
```
In the meantime I stumbled on another issue. I wanted to have colored text in cells using [`chalk`](https://www.npmjs.com/package/chalk), which adds this type of unicode chars: `\x1B[32m<text>\x1B[39m`. The coloring works alright, but it breaks the column width sync between headers & body:
Sample output when all the words in the first column are colored green with `chalk.green('text')`:

```
+----------------------+--------+---------+
|       Package        | Compat | Version |
+----------------------+--------+---------+
| coveralls  | >=6    | 3.1.1   |
| mocha      | N/A    | N/A     |
| nyc        | >=8.9  | 15.1.0  |
| release-it | >=10   | 14.11.8 |
+----------------------+--------+---------+
```

**Comment by AllMightySauron** [comment]
Weird, I have been using it without any issues in the exact same way. :) I'll look into it.

As for the second issue, this one I am already aware of. It as to do with the way I am calculating string sizes right now. Can you open a second issue so that I can correct it? I will do a quick test with `chalk` as well.

**Comment by webketje** [comment]
@AllMightySauron sure, btw update:
It actually works as expected for Node 12+, but not below (I reported the issue when testing on Node 10.22.1).
So the fix might be as simple as specifying  `"engines": { "node": ">=12" }` in package.json
Still using `require('package.json')` looks better than `fs.readFileSync(JSON.parse('package.json'))`.. Not sure which of the 2 is most performant, never did a benchmark

**Comment by AllMightySauron** [comment]
Corrected in 0.6.3.
